source/target/nordic/nrf5x/target_16.c

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
This file doesn't appear to be used anywhere.  Was the intent to remove the nrf51 folder and instead put the files in nrf5x?